.claude/skills/shared/scripts/gh_helpers.sh

```
@@ -0,0 +1,147 @@
+
+# Fetch PR reviews/comments
+# Usage: fetch_pr_reviews <pr_number>
+fetch_pr_reviews() {
+    local pr_number="$1"
+
+    if [ -z "$pr_number" ]; then
+        echo "Error: PR number required"
+        return 1
+    fi
+
+    local repo_info=$(get_repo_info)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)